.cargo/config_ci.toml

```diff
@@ -0,0 +1,53 @@
+# This config is used for the CI workflow.
+# Feel free to also use it locally for fast builds if you have the required setup!
+
+[unstable]
+codegen-backend = true
+
+[profile.dev]
+codegen-backend = "cranelift"
+
+# compile deps with llvm still for runtime perf
```

Comment:
This comment is from my `~/.cargo/config.toml`, so that doesn't make sense, sry.
However, I have experienced the test runner crash before when compiling everything with cranelift. Not sure if that's still the case, but I'd be careful about removing this line.